resources/sass/app.scss

```diff
@@ -337,14 +336,14 @@ dl.import-meta .download-csv {
             transition-property: background-color, border-color;
         }
         &:active {
-            background-color: $color-utility-red-30;
-            border-color: $color-utility-red-30;
+            background-color: $wikit-Button-primary-destructive-active-background-color;
+            border-color: $wikit-Button-primary-destructive-active-border-color;
             transition-property: background-color, border-color;
```

Comment:
We could use: $wikit-Button-transition-property

Reply:
Done in bba763be328c5e94d3eda0be8acab4717fca42cc.